pkg/kubeclient/kubelet_client.go

```diff
@@ -294,14 +295,15 @@ func TextToThrottlingMetricFamilies(data []byte) (map[string]*dto.MetricFamily,
 	return metricFamilies, nil
 }
 
-// get node single-core Frequency, in MHz
+// GetNodeCpuFrequency gets node single-core Frequency, in MHz
 func (client *KubeletClient) GetNodeCpuFrequency(node *v1.Node) (float64, error) {
 	ip, err := util.GetNodeIPForMonitor(node, types.KubeletSource)
 	if err != nil {
-		glog.Errorf("Failed to IP for node %s: %s", node.Name, err)
+		glog.Errorf("Failed to get IP for node %s: %s", node.Name, err)
 		return 0, err
 	}
-
+	os, arch := util.GetNodeOSArch(node)
+	glog.Infof("Node %s is running with %s/%s.", node.Name, os, arch)
```

Comment:
`GetNodeCpuFrequency()` is called repeatedly over the lifecycle of kubeturbo. Can this lead to log pollution, especially at this log level.

Reply:
Logging os/arch is requested by https://vmturbo.atlassian.net/browse/OM-73199, so I feel it is important. We need to log at every discovery, as the old logs may be recycled.